javascript/chronometer.js

```diff
@@ -1,35 +1,46 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    (this.currentTime = 0);
+    (this.intervalId = null);    
   }
 
   start(callback) {
-    // ... your code goes here
+    
+    this.intervalId = setInterval(() => {
+      this.currentTime += 1
```

Comment:
Aqui hizo falta agregar el callback
```suggestion
      this.currentTime += 1
      if(callback) callback()
```